fmriprep/cli/run.py

```diff
@@ -9,21 +9,34 @@
 
 import os
 import os.path as op
-import glob
+import logging
 import sys
 import uuid
+import warnings
 from argparse import ArgumentParser
 from argparse import RawTextHelpFormatter
 from multiprocessing import cpu_count
 from time import strftime
+from fmriprep.info import __version__
 
-from niworkflows.nipype import config as ncfg
+logging.addLevelName(25, 'INFO')  # Add a new level between INFO and WARNING
```

Comment:
Ah, gotcha. I thought it would override the regular "INFO". This is fine.

And yeah, verbosity flags for better control would be nice.